### PR TITLE
Feature/UI smartlib web/added languaje support fr, en ,es. and test

### PR DESCRIPTION
**Pull request by IsmaelP001**
Feature/UI smartlib web/added languaje support fr, en ,es. and test updated for new feature.

### CODE REVIEW

File: ui-smartlib-web/next.config.mjs [hunk 1]
```diff
@@ -1,6 +1,8 @@
+import createNextIntlPlugin from 'next-intl/plugin';
+ 
+const withNextIntl = createNextIntlPlugin();
+ 
 /** @type {import('next').NextConfig} */
-const nextConfig = {
-    output: "standalone",
-};
```
Comment (Maquendi) [inline ui-smartlib-web/next.config.mjs:4]: is there a reason why you removed this line ? "output: "standalone"

File: ui-smartlib-web/setupTests.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// setupTests.ts
```
Comment (Maquendi) [inline ui-smartlib-web/setupTests.ts]: please remove comments

File: ui-smartlib-web/src/app/[locale]/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+export default async function LocaleLayout({
+  children,
+  params: {locale}
+}: {
+  children: React.ReactNode;
+  params: {locale: string};
+}) {
+  // Providing all messages to the client
+  // side is the easiest way to get started
+  const messages = await getMessages();
```
Comment (Maquendi) [inline ui-smartlib-web/src/app/[locale]/layout.tsx:13]: please remove comments

File: ui-smartlib-web/src/app/[locale]/page.test.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import { render, screen } from "@testing-library/react";
+import { AbstractIntlMessages, NextIntlClientProvider } from "next-intl";
+import { expect, test } from "vitest";
+import Home from "./page";
+
+// reusable Helper to render different locales and messages
```
Comment (Maquendi) [inline ui-smartlib-web/src/app/[locale]/page.test.tsx]: please remove comments

File: ui-smartlib-web/src/app/[locale]/page.test.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+function renderWithIntl(locale: string, messages: AbstractIntlMessages) {
+  return render(
+    <NextIntlClientProvider locale={locale} messages={messages}>
+      <Home />
+    </NextIntlClientProvider>
+  );
+}
+
+// Test to check if the English JSON renders correctly
```
Comment (Maquendi) [inline ui-smartlib-web/src/app/[locale]/page.test.tsx]: please remove comments

File: ui-smartlib-web/src/app/[locale]/page.test.tsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      title: "Welcome to the Home Page",
+    },
+  };
+
+  renderWithIntl("en", messagesEn);
+
+  expect(screen.getByText("Welcome to the Home Page")).toBeInTheDocument();
+});
+
+// Test to check if the Frances JSON renders correctly
```
Comment (Maquendi) [inline ui-smartlib-web/src/app/[locale]/page.test.tsx]: please remove comments

File: ui-smartlib-web/src/app/[locale]/page.test.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    },
+  };
+
+  renderWithIntl("fr", messagesFr);
+
+  expect(screen.getByText("Bienvenue sur la page d'accueil")).toBeInTheDocument();
+});
+
+
+// Test to check if the spanish JSON renders correctly
```
Comment (Maquendi) [inline ui-smartlib-web/src/app/[locale]/page.test.tsx]: please remove comments

File: ui-smartlib-web/src/middleware.ts [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import { routing } from "@/i18n/routing";
+import createMiddleware from "next-intl/middleware";
+
+export default createMiddleware(routing);
+
+export const config = {
+  // Match only internationalized pathnames
```
Comment (Maquendi) [inline ui-smartlib-web/src/middleware.ts]: please remove comments

**Review by Maquendi — COMMENTED** [review]
Good job @IsmaelP001, please just check my comments and then we will proceed to merge.

File: ui-smartlib-web/next.config.mjs (hunk 1 above)
Comment (IsmaelP001) [reply ui-smartlib-web/next.config.mjs:4]: nope. my bad, i might have removed it when adding NextIntl support. by mistake.

**Review by Maquendi — APPROVED** [review]

**Pull request merged**